gcp-cups-connector-util/gcp-cups-connector-util.go

```diff
@@ -521,3 +561,41 @@ func showGCPPrinterStatus(context *cli.Context) {
 		fmt.Println(" ", state, ":", count)
 	}
 }
+
+// shareGCPPrinter shares a GCP printer
+func shareGCPPrinter(context *cli.Context) {
+	config := getConfig(context)
+	gcpConn := getGCP(config)
+
+	var role gcp.Role
+	switch strings.ToUpper(context.String("role")) {
+	case "USER":
+		role = gcp.User
+	case "MANAGER":
+		role = gcp.Manager
+	default:
+		fmt.Println("role should be user or manager.")
+		return
+	}
+
+	err := gcpConn.Share(context.String("printer-id"), context.String("email"),
+		role, context.Bool("skip_notification"))
+	if err != nil {
+		fmt.Printf("Failed to share GCP printer %s with %s: %s\n", context.String("printer-id"), context.String("email"), err)
+	} else {
+		fmt.Printf("Shared GCP printer %s with %s\n", context.String("printer-id"), context.String("email"))
+	}
+}
+
+// unshareGCPPrinter unshares a GCP printer.
+func unshareGCPPrinter(context *cli.Context) {
+	config := getConfig(context)
+	gcpConn := getGCP(config)
```

Comment:
@jacobmarble wasn't sure what to do here as other functions name this gcp which overwrites gcp/gcp.go namespace preventing me from using gcp.User / gcp.Manager constants. Should I rename this for all functions for consistency?